src/AzSK.ADO/Framework/Configurations/SVT/ControlSettings.json

```diff
@@ -296,7 +296,7 @@
       "Contributors": [
         "Manage build queue",
         "Stop builds",
-        "Override build checkIn validation",
+        "Override check-in validation by build",
```

Comment:
Are all these permissions being copied from the UI?

Reply:
yes